README.md

```diff
@@ -1 +1,29 @@
-# hw01
+# Biography of Anna Kenig-Ziesler 
+
+![Image of Me](IMG_0909_smaller.jpeg)
+
+## Academic Interests
+
+My name is Anna Kenig-Ziesler, and I am a third year in the College. I am majoring in Public Policy and minoring in Human Rights. I am really interested in better understanding how data can be used to inform public policy, so I am really excited to be taking this course! In the past, I have found that an inability to program can be quite limiting when it comes to data analysis, so I am looking forward to getting an introduction to computing! On campus, I am very involved in UChiVotes, and I am really passionate about getting out the vote this election. While I am unsure of what I want to do career-wise, I would love to work on environmental or civil rights issues in some capacity in the policy world. I have worked for non-profits the past few summers. This experience has really motivated me to get involved on the policy side, as I have realized that larger scale, policy solutions are necessary. I think that policy research is a big part of this, so I am excited to see how this class can open doors in terms of my research capabilities. 
+
+## Outside of Class
+
+Outside of class, a few of *my favorite things* are listed below: 
+
+* Soccer
+* Running
+* Biking
+* Cats (but I love dogs too)
+* Healthy (vegan) food
+* Dark chocolate
+* Outdoors
+
+I am a member of the Women's Soccer team ([Athletics Bio](https://athletics.uchicago.edu/sports/wsoc/2020-21/bios/kenig-ziesler_anna_u8hf)), and I also love to run. I am a little bit of a fitness freak, and I pretty much love any form of working out. I spend pretty much all of the (limited) free time that I have doing something active! One of my favorite things to do in Chicago is to run downtown with friends to get donuts. Our newest favorite place to go is [Beacon Doughnuts](https://www.beacondoughnuts.com), which is a small pop-up in Lincoln Park.  
```

Comment:
First off, I am the same! The links seem to be working great!

Reply:
Very detailed self-introduction! The headers, bullet points, italic feature and links all work well. All requirements of this homework were met, great work!